services/base/shared_libs/src/scanhub_libraries/models.py

```diff
@@ -286,7 +289,7 @@ class Config:
     site: str | None
     address: str | None
     creator: str
-    status: str
+    status: Literal["NEW", "UPDATED", "DELETED"]
```

Comment:
Is this registered by openapi? i.e. do we get some typescript status model or do you need to pass the correct string?

Reply:
Yes, it is registered :)